File: in_go/src/pattern_8.go
package src

import "fmt"

func Pattern_8() {
	row := 5
	column := 9
	symbol := 1

	for i := 0; i < row; i++ {
		symbol = 1
		for j := 0; j < column; j++ {
			if j >= row-1-i && j <= row-1+i {
				fmt.Print(symbol)
				if j < row-1 {
					symbol++
				} else {
					symbol--
				}
			} else {
				fmt.Print(" ")
			}
		}
		fmt.Println()
	}
}

File: in_typescript/src/Pattern_8.ts
export default function pattern_8() {
  let row = 5;
  let column = 9;

  let symbol = 1;
  for (let i = 0; i < row; i++) {
    symbol = 1;
    for (let j = 0; j < column; j++) {
      if (j >= row - 1 - i && j <= row - 1 + i) {
        process.stdout.write(`${symbol}`);
        if (j < row - 1) {
          symbol++;
        } else {
          symbol--;
        }
      } else {
        process.stdout.write(" ");
      }
    }
    process.stdout.write("\n");
  }
}
